rpm/generic/spl-dkms.spec.in

```diff
@@ -7,6 +7,16 @@
 %define module  @PACKAGE@
 %define mkconf  scripts/dkms.mkconf
 
+# Python permits the !/usr/bin/python shebang for scripts that are cross
+# compatible between python2 and python3, but Fedora 28 does not.  Fedora
+# wants us to choose python3 for cross-compatible scripts.  Since we want
+# to support python2 and python3 users, exclude our scripts from Fedora 28's
+# RPM build check, so that we don't get a bunch of build warnings.
+#
+# Details: https://github.com/zfsonlinux/zfs/issues/7360
+#
+%global __brp_mangle_shebangs_exclude_from      splslab.py
```

Comment:
`splslab.py` shouldn't be included in the `spl-dkms` package so this shouldn't be needed.

Reply:
Strangely, rpm-dkms does need it (but not rpm-kmod):
```
$ make rpm-dkms
make[1]: Leaving directory '/home/hutter/spl'
make  pkg="spl-dkms" \
	def='--define "_without_debug 1" --define "_without_debug_kmem 1" --define "_without_debug_kmem_tracking 1" ' rpm-common
make[1]: Entering directory '/home/hutter/spl'
make[2]: Entering directory '/home/hutter/spl'
make[2]: Leaving directory '/home/hutter/spl'
Installing spl-dkms-0.7.0-1.fc28.src.rpm
Executing(%prep): /bin/sh -e /tmp/spl-build-hutter-Vibrj9sa/TMP/rpm-tmp.3tdzlK
+ umask 022
+ cd /tmp/spl-build-hutter-Vibrj9sa/BUILD
+ cd /tmp/spl-build-hutter-Vibrj9sa/BUILD
+ rm -rf spl-0.7.0
+ /usr/bin/gzip -dc /tmp/spl-build-hutter-Vibrj9sa/SOURCES/spl-0.7.0.tar.gz
+ /usr/bin/tar -xof -
+ STATUS=0
+ '[' 0 -ne 0 ']'
+ cd spl-0.7.0
+ /usr/bin/chmod -Rf a+rX,u+w,g-w,o-w .
+ exit 0
Executing(%build): /bin/sh -e /tmp/spl-build-hutter-Vibrj9sa/TMP/rpm-tmp.bkwWF1
+ umask 022
+ cd /tmp/spl-build-hutter-Vibrj9sa/BUILD
+ cd spl-0.7.0
+ scripts/dkms.mkconf -n spl -v 0.7.0 -f dkms.conf
+ exit 0
Executing(%install): /bin/sh -e /tmp/spl-build-hutter-Vibrj9sa/TMP/rpm-tmp.2ryx2i
+ umask 022
+ cd /tmp/spl-build-hutter-Vibrj9sa/BUILD
+ '[' /tmp/spl-build-hutter-Vibrj9sa/BUILDROOT/spl-dkms-0.7.0-1.fc28.x86_64 '!=' / ']'
+ rm -rf /tmp/spl-build-hutter-Vibrj9sa/BUILDROOT/spl-dkms-0.7.0-1.fc28.x86_64
++ dirname /tmp/spl-build-hutter-Vibrj9sa/BUILDROOT/spl-dkms-0.7.0-1.fc28.x86_64
+ mkdir -p /tmp/spl-build-hutter-Vibrj9sa/BUILDROOT
+ mkdir /tmp/spl-build-hutter-Vibrj9sa/BUILDROOT/spl-dkms-0.7.0-1.fc28.x86_64
+ cd spl-0.7.0
+ '[' /tmp/spl-build-hutter-Vibrj9sa/BUILDROOT/spl-dkms-0.7.0-1.fc28.x86_64 '!=' / ']'
+ rm -rf /tmp/spl-build-hutter-Vibrj9sa/BUILDROOT/spl-dkms-0.7.0-1.fc28.x86_64
+ mkdir -p /tmp/spl-build-hutter-Vibrj9sa/BUILDROOT/spl-dkms-0.7.0-1.fc28.x86_64/usr/src/
+ cp -rf /tmp/spl-build-hutter-Vibrj9sa/BUILD/spl-0.7.0 /tmp/spl-build-hutter-Vibrj9sa/BUILDROOT/spl-dkms-0.7.0-1.fc28.x86_64/usr/src/
+ /usr/lib/rpm/find-debuginfo.sh -j4 --strict-build-id -m -i --build-id-seed 0.7.0-1.fc28 --unique-debug-suffix -0.7.0-1.fc28.x86_64 --unique-debug-src-base spl-dkms-0.7.0-1.fc28.x86_64 --run-dwz --dwz-low-mem-die-limit 10000000 --dwz-max-die-limit 110000000 -S debugsourcefiles.list /tmp/spl-build-hutter-Vibrj9sa/BUILD/spl-0.7.0
+ /usr/lib/rpm/check-buildroot
+ /usr/lib/rpm/redhat/brp-ldconfig
/sbin/ldconfig: Warning: ignoring configuration file that cannot be opened: /etc/ld.so.conf: No such file or directory
+ /usr/lib/rpm/brp-compress
+ /usr/lib/rpm/brp-strip-static-archive /usr/bin/strip
+ /usr/lib/rpm/brp-python-bytecompile /usr/bin/python 1
/tmp/spl-build-hutter-Vibrj9sa/BUILDROOT/spl-dkms-0.7.0-1.fc28.x86_64/usr/src/spl-0.7.0/cmd/splslab/splslab.py
DEPRECATION WARNING: python2 invoked with /usr/bin/python.
    Use /usr/bin/python3 or /usr/bin/python2
    /usr/bin/python will be removed or switched to Python 3 in the future.
    If you cannot make the switch now, please follow instructions at https://fedoraproject.org/wiki/Changes/Avoid_usr_bin_python_in_RPM_Build#Quick_Opt-Out
DEPRECATION WARNING: python2 invoked with /usr/bin/python.
    Use /usr/bin/python3 or /usr/bin/python2
    /usr/bin/python will be removed or switched to Python 3 in the future.
    If you cannot make the switch now, please follow instructions at https://fedoraproject.org/wiki/Changes/Avoid_usr_bin_python_in_RPM_Build#Quick_Opt-Out
+ /usr/lib/rpm/brp-python-hardlink
+ /usr/lib/rpm/redhat/brp-mangle-shebangs '' ''
*** WARNING: mangling shebang in /usr/src/spl-0.7.0/cmd/splslab/splslab.py from #!/usr/bin/python to #!/usr/bin/python2. This will become an ERROR, fix it manually!
```

Reply:
Strange, well it's harmless enough.